.github/workflows/build.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - uses: actions/checkout@v3
 
       - name: Install Dependencies
-        run: npm ci
+        run: npm install
```

Comment:
why dis?

Reply:
The unit tests were failing at the "Install dependencies" step each time when I opened the pull request, Glen suggested using npm install instead and it seemed to resolve it